rp2040-hal-examples/src/bin/spi_eh_bus.rs

```diff
@@ -0,0 +1,307 @@
+//! # SPI Bus example
+//!
+//! This application demonstrates how to construct a simple SPI Driver and
+//! configure rp2040-hal's SPI peripheral to access it by utilising
+//! [`ExclusiveDevice`] from [`embedded-hal-bus`].
+//!
+//! [`ExclusiveDevice`]:
+//!     https://docs.rs/embedded-hal-bus/latest/embedded_hal_bus/spi/struct.ExclusiveDevice.html
+//! [`embedded-hal-bus`]: https://crates.io/crates/embedded-hal-bus
+//!
+//! It may need to be adapted to your particular board layout and/or pin
+//! assignment.
+//!
+//! See the top-level `README.md` file for Copyright and license details.
+//!
+//! ## Glossary
+//!
+//! * *SPI Bus* - a shared bus consisting of three shared wires (Clock, COPI and
+//!   CIPO) and a unique 'Chip Select' wire for each device on the bus. An *SPI
```

Comment:
I agree with avoiding the terms master/slave in technical descriptions. However we still use MOSI/MISO in rp2040-hal and rp2350-hal docs. Shouldn't we use the terms consistently?

Reply:
We should be consistent. Using the acronym but not expanding it is perhaps the first step - then renaming it everywhere.

Reply:
Tracking that in https://github.com/rp-rs/rp-hal/issues/846